include/rocksdb/listener.h

```diff
@@ -148,10 +148,22 @@ struct TableFileDeletionInfo {
   Status status;
 };
 
+enum class FileOperationType {
+  kRead,
+  kWrite,
+  kTruncate,
+  kClose,
+  kFlush,
+  kSync,
+  kFsync,
+  kRangeSync
+};
+
 struct FileOperationInfo {
   using TimePoint = std::chrono::time_point<std::chrono::system_clock,
                                             std::chrono::nanoseconds>;
 
+  FileOperationType type;
```

Comment:
Since we are on this. Would it be possible to update the signature of `FileOperationInfo(...)` so that it takes additional arguments `status` and `type`?

Reply:
done.